api/docs/building.dox

```diff
@@ -286,7 +286,7 @@ To build a client, again use the toolchain file, as well as pointing at a Dynamo
 
```

Comment:
As discussed offline, please test the ci-package action against this branch and point at the result in the PR description (it should then be the official weekly cronbuild too).

Reply:
Done.